### PR TITLE
Should not be adding TM comments automatically

### PR DESCRIPTION
**Pull request by Cyberboss**
Removes Fast Add and Fast Update, auto genned test merge comments. Fast update is replaced with the existing test merge comment.

### CODE REVIEW

File: src/components/utils/TestMergeRow.tsx [hunk 1]
```diff
@@ -194,7 +194,7 @@ export default function TestMergeRow({
                                                 e.shiftKey
                                                     ? onSelectCommit(
                                                           pr.head,
-                                                          "No comment set - Fast Update"
+                                                          testmergeinfo?.comment ?? null
```
Comment (alexkar598) [inline src/components/utils/TestMergeRow.tsx:197]: What's the point of this?

Comment (Cyberboss) [reply src/components/utils/TestMergeRow.tsx:197]: This keep the existing comment if there is one

Comment (alexkar598) [reply src/components/utils/TestMergeRow.tsx:197]: No, what's the point of the ?? null

Comment (Cyberboss) [reply src/components/utils/TestMergeRow.tsx:197]: `comment?: string | null` into a field that is `string | null`

**Pull request merged**